best-practices/haproxy-best-practices.md

```diff
@@ -41,19 +41,20 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
 
 ### 依赖软件
 
-根据官方文档，对操作系统和依赖包有以下建议，如果通过 yum 源部署安装 HAProxy 软件，依赖包无需单独安装。
+根据 HAProxy 官方文档，对操作系统和依赖包有以下建议，如果通过 yum 源部署安装 HAProxy 软件，依赖包无需单独安装。
 
 #### 操作系统
 
-| 操作系统版本               | 架构                                       |
-|:-------------------------|:------------------------------------------|
-| Linux 2.4                | x86、x86_64、Alpha、SPARC、MIPS 和 PA-RISC  |
-| Linux 2.6 或 3.x         | x86、x86_64、ARM、SPARC 和 PPC64            |
-| Solaris 8 或 9           | UltraSPARC II 和 UltraSPARC III            |
-| Solaris 10               | Opteron 和 UltraSPARC                      |
-| FreeBSD 4.10 ~ 10        | x86                                        |
-| OpenBSD 3.1 及以上版本     | i386、AMD64、macppc、Alpha 和 SPARC64       |
-| AIX 5.1 ~ 5.3            | Power™                                     |
+| Linux 操作系统平台       | 版本         |
```

Comment:
| OS platform      | Version                    |
|----------------|--------------------|
| Linux kernel      | 2.6.28 and above   |
| Solaris               | 10 and above          | 
| FreeBSD            | 10 and above         | 
| DragonFlyBSD  | 4.3 and above        | 
| NetBSD             | 8 and above            |
| Mac OS/X          | N/A                           | 
| OpenBSD          | 6.3 and above          | 
| AIX                     | 5.1 and 5.2 and 7.2  | 
| Cygwin              | N/A                            | 
| Haiku                 | N/A                            |

> **注意：**
>
> - 其他操作系统支持情况，详见 [HAProxy 文档](https://github.com/haproxy/haproxy/blob/master/INSTALL)。